benchmarking/commons/benchmark_definitions/yahpo.py

```diff
@@ -435,32 +473,1524 @@ def yahpo_rbv2_svm_benchmark(dataset_name, metric, mode="max"):
+            name,
+            metric,
+            mode,
+            restrict_fidelities=restrict_fidelities,
+            max_wallclock_time=yahpo_rbv2_max_wallclock_time[
+                f"{method}-{metric}-{name}"
+            ],
+        )
+        for metric, mode in yahpo_rbv2_metrics
+        for name in yahpo_rbv2_instances[method][metric]
+    }
+
+
+def yahpo_rbv2_selected_benchmark_definitions(
+    method, restrict_fidelities=False
+) -> dict:
+    return {
+        benchmark_name("rbv2", method, metric, name): yahpo_rbv2_benchmark[method](
+            name,
+            metric,
+            mode,
+            restrict_fidelities=restrict_fidelities,
+            max_wallclock_time=yahpo_rbv2_max_wallclock_time[
+                f"{method}-{metric}-{name}"
+            ],
+        )
+        for metric, mode in yahpo_rbv2_metrics
+        for name in yahpo_rbv2_selected_instances[method][metric]
+    }
+
+
+def yahpo_rbv2_rpart_benchmark_definitions(restrict_fidelities=False) -> dict:
+    return yahpo_rbv2_benchmark_definitions("rpart", restrict_fidelities)
+
+
+def yahpo_rbv2_rpart_selected_benchmark_definitions(restrict_fidelities=False) -> dict:
+    return yahpo_rbv2_selected_benchmark_definitions("rpart", restrict_fidelities)
+
+
+def yahpo_rbv2_glmnet_benchmark_definitions(restrict_fidelities=False) -> dict:
+    return yahpo_rbv2_benchmark_definitions("glmnet", restrict_fidelities)
+
+
+def yahpo_rbv2_glmnet_selected_benchmark_definitions(restrict_fidelities=False) -> dict:
+    return yahpo_rbv2_selected_benchmark_definitions("glmnet", restrict_fidelities)
+
+
+def yahpo_rbv2_ranger_benchmark_definitions(restrict_fidelities=False) -> dict:
+    return yahpo_rbv2_benchmark_definitions("ranger", restrict_fidelities)
+
+
+def yahpo_rbv2_ranger_selected_benchmark_definitions(restrict_fidelities=False) -> dict:
+    return yahpo_rbv2_selected_benchmark_definitions("ranger", restrict_fidelities)
+
+
+def yahpo_rbv2_xgboost_benchmark_definitions(restrict_fidelities=False) -> dict:
+    return yahpo_rbv2_benchmark_definitions("xgboost", restrict_fidelities)
+
+
+def yahpo_rbv2_xgboost_selected_benchmark_definitions(
+    restrict_fidelities=False,
+) -> dict:
+    return yahpo_rbv2_selected_benchmark_definitions("xgboost", restrict_fidelities)
+
+
+def yahpo_rbv2_svm_benchmark_definitions(restrict_fidelities=False) -> dict:
+    return yahpo_rbv2_benchmark_definitions("svm", restrict_fidelities)
+
+
+def yahpo_rbv2_svm_selected_benchmark_definitions(restrict_fidelities=False) -> dict:
+    return yahpo_rbv2_selected_benchmark_definitions("svm", restrict_fidelities)
+
+
+def yahpo_rbv2_aknn_benchmark_definitions(restrict_fidelities=False) -> dict:
+    return yahpo_rbv2_benchmark_definitions("aknn", restrict_fidelities)
+
+
+def yahpo_rbv2_aknn_selected_benchmark_definitions(restrict_fidelities=False) -> dict:
+    return yahpo_rbv2_selected_benchmark_definitions("aknn", restrict_fidelities)
```

Comment:
This is similar to https://github.com/awslabs/syne-tune/pull/396/files#r1003251413